ocs_sample_library_preview/Asset/AssetRule.py

```diff
@@ -0,0 +1,97 @@
+import json
+
+
+class AssetRule(object):
+    """OCS Asset Rule definition"""
+
+    def __init__(self, id: str = None, name: str = None, description: str = None):
+        """
+        :param id: required
+        :param name: required
+        :param description: not required
+        """
+        self.Id = id
+        self.Name = name
+        self.Description = description
```

Comment:
I assume this class implementation as a stub with only these properties was the quickly spin it up to expose the ACL? Should we note somewhere in the class that it's only a stub for now and may be filled in later with actual AssetRule CRUD operations?

Reply:
@dcbroad3 Had a similar comment, but the idea is that implementing other routes falls outside the scope of implementing security. Rather than waiting for these routes to come into scope, I added them as stubs so that their security could be managed. I could leave a comment in the classes where this is the case if that would be helpful.

Reply:
I think a quick comment wouldn't hurt. especially as an opportunity to say to reach out to samples or to uservoice to express interest in them being built out

Reply:
comments added to stubs